### ISSUE TITLE
2.1.0 update breaking builds

### ISSUE THREAD

**Issue by jkomusin** [GoogleChromeLabs/preload-webpack-plugin#45]
After 2.1.0 was published, our builds started breaking with the following:

```
ERROR in   TypeError: Object.values is not a function

  - index.js:80 extractedChunks.filter.chunk
    [app]/[preload-webpack-plugin]/index.js:80:93
```

Is this a config problem on our end, or an edge case missed in this new version?

EDIT: Sorry, looks like Object.values is only available by default in node v7+. Feel free to close this, but it may be nice to add node v7 as a dependency.

**Comment by jeffposnick** [comment]
Sorry about that—I see now that there's no minumum node version defined in `package.json`, so we should definitely strive for compatibility with older releases.

`Object.values` was introduced in 7.x, it looks like: http://node.green/#ES2017-features-Object-static-methods-Object-values

I'll put together a quick 2.1.1 release that pulls in https://www.npmjs.com/package/object.values and longer term I think we'll need to document a minimum supported version of node and use babel to transpile against that as a preset.

**Comment by jeffposnick** [comment]
Can you switch your dependency in `package.json` to

```
"preload-webpack-plugin": "git+https://github.com/GoogleChromeLabs/preload-webpack-plugin.git#object-values-shim"
```

and confirm that fixes things for you? I'll published that as 2.1.1 assuming it does.

**Comment by jkomusin** [comment]
Yup, that seems to have fixed it for us back in node 6 land! Thanks for the quick response 😄
